addons/mail/static/src/components/rtc_call_viewer/rtc_call_viewer.xml

```diff
@@ -3,25 +3,26 @@
 
 <t t-name="mail.RtcCallViewer" owl="1">
     <t t-if="rtcCallViewer">
-        <div class="o_RtcCallViewer" t-att-class="{ 'o-fullScreen': rtcCallViewer.isFullScreen, 'o-isMinimized': rtcCallViewer.isMinimized }" t-attf-class="{{ className }}" t-ref="root">
+        <div class="o_RtcCallViewer d-flex flex-column align-items-center justify-content-center bg-dark" t-att-class="{'o-fullScreen fixed-top vw-100 vh-100': rtcCallViewer.isFullScreen, 'o-isMinimized h-25': rtcCallViewer.isMinimized, 'h-50': !rtcCallViewer.isMinimized, 'position-relative': !rtcCallViewer.isFullScreen }" t-attf-class="{{ className }}" t-ref="root">
             <!-- Used to make the component depend on the window size and trigger an update when the window size changes. -->
             <t t-set="dummy" t-value="messaging and messaging.device and messaging.device.globalWindowInnerHeight and messaging.device.globalWindowInnerWidth"/>
 
             <!-- Call members display -->
-            <div class="o_RtcCallViewer_participantContainer" t-on-click="rtcCallViewer.onClick" t-on-mousemove="rtcCallViewer.onMouseMove">
+            <div class="o_RtcCallViewer_participantContainer d-flex justify-content-between w-100 h-100 overflow-hidden" t-on-click="rtcCallViewer.onClick" t-on-mousemove="rtcCallViewer.onMouseMove">
                 <t t-if="rtcCallViewer.layout !== 'tiled'">
-                    <div class="o_RtcCallViewer_mainParticipantContainer justify-content-center mw-100 mh-100">
+                    <div class="o_RtcCallViewer_mainParticipantContainer d-flex flex-grow-1 justify-content-center mw-100 mh-100">
                         <t t-if="rtcCallViewer.mainParticipantCard">
                             <RtcCallParticipantCard className="'o_RtcCallViewer_participantCard w-100'" record="rtcCallViewer.mainParticipantCard"/>
                         </t>
                     </div>
                 </t>
                 <t t-if="rtcCallViewer.layout !== 'spotlight'">
                     <div
-                        class="o_RtcCallViewer_grid"
+                        class="o_RtcCallViewer_grid d-flex align-items-center h-100"
                         t-att-class="{
-                            'o-isTiled': rtcCallViewer.layout === 'tiled',
-                            'o-isSidebar': rtcCallViewer.layout === 'sidebar',
+                            'o-isTiled justify-content-center w-100': rtcCallViewer.layout === 'tiled',
+                            'flex-wrap': rtcCallViewer.layout !== 'tiled',
```

Comment:
I think there's an issue here. According to the css you removed at lines 43 to 59, 
- `flex-wrap` should be triggered with `o-isTiled`.
- `flex-column` should be triggered in all cases except when `o-isTiled` is triggered.